.github/workflows/merge-flow.yaml

```diff
@@ -159,6 +164,12 @@ jobs:
       - name: Continue after merge conflict
         if: ${{ steps.merge.outputs.MERGE_CONFLICT == 'true' }}
         run: git merge --continue
+      - name: Add VERSION file
+        if: ${{ inputs.version-file-sed-expr != '' }}
+        run: |
+          echo ${{ steps.upstream.outputs.release }} | sed -e ${{ inputs.version-file-sed-expr }} > VERSION
```

Comment:
I'll see how to avoid duplicating the commands under `If you wish to perform this manually...`